Pages/Index/Index.cshtml

```diff
@@ -1,17 +1,22 @@
+@page "/"
```

Comment:
This is how we can override the route conventions so this emulates the `MapGet("/"` we previously had